tools/tsconfig.tools.json

```diff
@@ -5,8 +5,8 @@
     "rootDir": ".",
     "module": "commonjs",
     "target": "es5",
-    "types": ["node"]
+    "types": ["node"],
+    "importHelpers": false
```

Comment:
question: Interesting. Is this to remove dependencies with `tslib`?

Reply:
I don't know, it's just a default file from a fresh Nx 13.1 workspace. In later Nx versions, the tools folder might even be removed, so I didn't want anything custom here.